crates/services/sync/src/import.rs

```diff
@@ -118,8 +123,11 @@ where
         &self,
         shutdown: &mut StateWatcher,
     ) -> anyhow::Result<bool> {
+        let start = ServiceMetrics::instant();
+
         self.import_inner(shutdown).await?;
 
+        IMPORT_METRICS.observe(start);
```

Comment:
Each service may have the instance inside of `ServiceRunner` and you don't need to use lazy=)